.gitignore

```diff
@@ -11,3 +11,4 @@ testing/
 /.vs
 /out/build
 /CMakeSettings.json
+/out/mytest
```

Comment:
what's that?

Reply:
stuff in visual studio, which puts all kinds of stuff into out.